spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/web/TraceFilterTests.java

```diff
@@ -110,6 +110,21 @@ public void startsNewTrace() throws Exception {
 	}
 
 	@Test
+	public void startsNewTraceWithParentIdInHeaders() throws Exception {
+		this.request = builder()
+				.header(Span.SPAN_ID_NAME, Span.toHex(1L))
```

Comment:
similarly `idToHex` re-inforces the intent and also why it is on span. ex we aren't turning a span into hex, we are turning an id into hex.